src/sage/rings/qqbar.py

```diff
@@ -4485,6 +4485,26 @@ def as_number_field_element(self, minimal=False, embedded=False, prec=53):
         """
         return number_field_elements_from_algebraics(self, minimal=minimal, embedded=embedded, prec=prec)
 
+    def is_integral(self):
+        r"""
+        Determine if a number is an algebraic integer.
+
```

Comment:
This description does not read well. The method works with the given object, namely, ``self``.  It is not just "a number". I suggest
```
Check if this number is an algebraic integer.
```
or
```
Return ``True`` if this number is an algebraic integer.
```

Reply:
Okay. 

(Result of mindlessly copying from the other method I guess. In `number_field_element.pyx`:

```python
    def is_integral(self):
        r"""
        Determine if a number is in the ring of integers of this number
        field.

```
)

Reply:
Fixed, thanks for the review.